hub-templates/base-hub/values.yaml

```diff
@@ -131,9 +132,13 @@ jupyterhub:
         - ports:
             - port: 443
               protocol: TCP
-  auth:
-    type: dummy
   hub:
+    config:
+      JupyterHub:
+        authenticator_class: oauthenticator.generic.GenericOAuthenticator
+    image:
+      name: quay.io/2i2c/pilot-hub
```

Comment:
I was experimenting with this hub image when I pushed it. I think we can remove this and use the default image for now.